velox/docs/functions/presto/string.rst

```diff
@@ -153,6 +153,8 @@ String Functions
     Returns the position of the N-th ``instance`` of ``substring`` in ``string``.
     ``instance`` must be a positive number.
     Positions start with ``1``. If not found, ``0`` is returned.
+    It takes into account overlapping strings when counting occurrences.
```

Comment:
Would you generate the docs and make sure they look good.

I think you need :: after the description and a new line:

```
It takes into account overlapping strings when counting occurrences.::

        SELECT strpos('aaa', 'aa', 2); -- 2
```

Reply:
i will add that before landing this time.

Reply:
@mbasmanova @laithsakka Thanks for your review. I have updated.